### ISSUE TITLE
Beef up class-quest reward equipment

### ISSUE THREAD

**Issue by rmsr** [sneezymud/sneezymud#90]
A person wrote on Facebook, "Quest items need to be reviewed to see if they fit in their expected levels and the inflation that the EQ in general has seen. I'm a fan of the deikhan weapons, but a lot of folks skip them in favorite of the "better" available weapons from the shops. Same with monk sashes. I really feel like quest items should be an advantage to their classes."

**Comment by nenbeely** [comment]
this is a great point - i was recently pleasantly surprised to find that the warrior scabbard quest equipment is legit usable late-game, but in most other cases the class quest gear is underpowered.

**Comment by Aion2501** [comment]
Agreed. Some quest items need to become more relevant. I would be happy to rewrite them. Need to get that handy builder interface up or do something so that I don't have to input/edit mobs and items in binary.  Devs already know what Im talking about here.
